.github/workflows/tox-test.yml

```diff
@@ -3,8 +3,8 @@ name: Tox tests
 on: [push, pull_request]
 
 jobs:
-  py27:
-    runs-on: ubuntu-20.04
+  py38:
```

Comment:
I don't think you needed to add this, because the `coverage` job already uses python3.  Dropping python2 doesn't require adding any new python3 jobs for this project.

Also, I believe this is the cause of the CI failure, because the dependency resolution for python 3.8 and 3.9 are probably not giving the same results.

Can you drop this please?  CI will then hopefully pass.